### PR TITLE
Ft js object

### PR DESCRIPTION
**Pull request by 41hulk**
## What does this PR do?
- [ ] Implement only a basic UI with plain HTML
- [ ] Create a collection that keeps a list of books.
- [ ] Create a function to add a new book to the collection, with title and author.
- [ ] Create a function to remove a book from the collection.
- [ ] Display all books saved in the collection in the top part of the page.
- [ ] When the user clicks the "Add" button, add new book is added to the collection, and the new book is displayed on the page.
- [ ] When a user clicks the "Remove" button, the correct book is removed from the collection and the correct book disappears from the page.

### CODE REVIEW

File: final.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+function Book(title, author, id) {
```
Comment (Meltrust) [inline final.js:1]: According to the project requirements:

image

Therefore:

You are indeed saving your books on localStorage, but not deleting them or displaying them when the page reloads.

- [x] Please, kindly do make sure that the correct data is preserved and updated in the browser's memory by using localStorage, and that the correct books display when the page is reloaded.


If you don't know how to do it yet, here is the [lesson](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md) that covers it (don't worry it's easier than it looks!).  

That way, we will meet this particular requirement and move closer to approval 💪

File: final.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  const id = Math.random()
+    .toString(36)
+    .replace(/[^a-z]+/g, '')
+    .substr(0, 5);
+  // Instantiate Book
+  const book = new Book(title, author, id);
+  // Display Book in UI
+  myBooks = myBooks.concat(book);
+  addBook(book);
+  console.log(myBooks);
```
Comment (Meltrust) [inline final.js]: According to Javascript's best practices:

image

And according to the project requirements themselves:

image

Therefore:

- [x] Please, kindly remove this debugging `console.log` so the repo is kept clean. 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @41hulk, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Good job! You are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @41hulk, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**